doc/source/dev/gitwash/configure_git.rst

```diff
@@ -100,13 +111,16 @@ like this::
           stat = status -a
           br = branch
           wdiff = diff --color-words
+          show-conflicts = !git --no-pager diff --name-only --diff-filter=U
 
 Editor
 ------
 
-You may also want to make sure that your editor of choice is used ::
+You may also want to make sure that your favorite editor is used:
+
+.. code-block:: bash
 
-  git config --global core.editor vim
+   $ git config --global core.editor nano
```

Comment:
:-) At least with `nano` we're "neutral" in the [editor war](https://en.wikipedia.org/wiki/Editor_war).